### PR TITLE
add gsc_signature tool to the galaxysinglecell release PR

### PR DESCRIPTION
**Pull request by drosofff**

### CODE REVIEW

**Comment by drosofff** [comment]
@bellenger-l We need to review a couple of plot titles, that are a bit cryptic, if not misleading...

**Comment by bellenger-l** [comment]
> @bellenger-l We need to review a couple of plot titles, that are a bit cryptic, if not misleading...

Ok !

**Comment by drosofff** [comment]
@bellenger-l OK lea, I put some comments in review about the graph titles

File: tools/gsc-signature_score/signature_score.R [hunk 1]
```diff
@@ -192,21 +177,16 @@ ggplot(data.frame(density_score[1:2]), aes(x, y, fill = ifelse(x < mean(score$sc
...
-p_gene <- ggplot(metadata_filtered, aes(Signature_category, nGene)) +
-  geom_violin(aes(fill = Signature_category), alpha = .5, trim = F, show.legend = F) +
+p_gene <- ggplot(signature_output, aes(rate, nGenes)) +
+  geom_violin(aes(fill = rate), alpha = .5, trim = F, show.legend = F) +
   scale_fill_manual(values=c("#ff0000", "#08661e")) +
   geom_jitter() + labs(y = "Number of detected genes", x = "Signature")
 
-p_counts <- ggplot(metadata_filtered, aes(Signature_category, total_counts)) +
-  geom_violin(aes(fill = Signature_category), alpha = .5, trim = F, show.legend = F) +
+p_counts <- ggplot(signature_output, aes(rate, total_counts)) +
```
Comment (drosofff) [inline tools/gsc-signature_score/signature_score.R]: I do not understand the title of this graph. What does that mean ?

File: tools/gsc-signature_score/signature_score.R [hunk 2]
```diff
@@ -185,41 +170,33 @@ ggplot(data.frame(density_score[1:2]), aes(x, y, fill = ifelse(x < mean(score$sc
   scale_fill_manual(values=c("#ff0000", "#08661e")) +
   ylim(0, max(density_score$y)) +
   labs(
-    title = "Score density",
+    title = "Distribution of Cell signature scores",
```
Comment (bellenger-l) [inline tools/gsc-signature_score/signature_score.R]: "Density of Cell signature scores" ?

File: tools/gsc-signature_score/signature_score.R (hunk 1 above)
Comment (bellenger-l) [reply tools/gsc-signature_score/signature_score.R]: Ok pour "Influence of library sequencing depth on cell signature scores"

**Pull request merged**